src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -7,13 +7,15 @@
 
 # system modules
 import time
+import datetime
```

Comment:
Given that datetime usage is very basic in this module, why not to simply stick to the `time` module that is already imported here?